views/showcase.jade

```diff
@@ -0,0 +1,18 @@
+extends layout
+
+block content
+    h2 Showcase
+    - each item in config.showcase
+        - if (!item.latest)
+            div.col-md-6.text-center
```

Comment:
This doesn't need the `div`, it's automatically added and should have an offset, i.e. `.col-md-6.col-md-offset-3.text-center`.